u3-views/views/partials/video-card.hbs

```diff
@@ -0,0 +1,8 @@
+<a class='home-card' href='/videos/{{tease.id}}'>
+  <div>
+    <img src='/images/{{tease.id}}.webp' />
+    <p>{{tease.duration}}</p>
+  </div>
+  <p>{{tease.title}}</p>
+  <p class='stats'>{{tease.viewsShort}} views · {{tease.dateRelative}}</p>
+</a>
```

Comment:
รหัสนี้ใช้งานได้ แต่อาจทำให้สับสน
- มุมมอง `video-tease` แสดงวิดีโอ `tease`
- มุมมอง `video-card` จะแสดง `card`

ฉันแนะนำให้ตั้งชื่อตัวแปรต่างกันเพื่อหลีกเลี่ยงความสับสน! ในมุมมองนี้และใน `video-home` (ที่คุณรวมมุมมองไว้ด้วย) ฉันขอแนะนำให้ใช้ตัวแปรที่เรียกว่า `video` หรือ `card`

-----

This code works, but it might be confusing.
- The `video-tease` view displays a video `tease`.
- The `video-card` view displays a video `card`.

I recommend naming the variable differently to avoid confusion! In this view, and in `video-home` (where you include the view), I recommend using a variable called either `video` or `card`.

Reply:
I'm fix it now teacher, please check it.

Reply:
Thank you! I will re-review tomorrow.